### ISSUE TITLE
Home page when opening the min-browser

### ISSUE THREAD

**Issue by carlosnizer** [minbrowser/min#762]
Hello
How do I change the start page when opening the min-browser?
I'm loving the browser, fast, clean, no frills.
But I wanted on the homepage, when I opened the min-browser, that I opened my page (with my links, in an https address, on my server.).
I found the answer # 364. I did as stated in that answer, but it did not change. It keeps opening the min tour page. I wanted to replace so always open my homepage from my server.
Thanks!!!

**Comment by PalmerAL** [comment]
You shouldn't be seeing the tour page after the first startup regardless of whether you make the changes from 364; there's a couple of things I can think of that might be causing this:

* The startup page is loaded as a normal tab, so it won't stop showing up until you close it, or click the "start browsing" button.
* Open sessions are only saved every 12.5 seconds, or when you switch tabs, so if you're closing the browser immediately after opening it, the fact that you loaded a new webpage may not be saved.
* It's possible there's an error happening that's preventing your session from being saved, can you check in the console (developer > inspect browser > console tab) and see if there are any errors?

**Comment by carlosnizer** [comment]
Hello
My intention is that anyone who has access to my service from my server uses the browser-min. Among the services of the server there is a stream service, web stream. Many report problems but this is due to the use of various browsers, with several installed plugins, extensions, spams, etc ....
The mim-browser works 100% and gives no chance for them to install unnecessary things.
I wanted to package the min-browser for anyone who downloaded it on my server to install it, and when opened it, it would open the main server page, which contains the links and services, since first time. obviously it would be maintained everything concerning your creation, 'about' ..., etc ...
Or, keep the page tour-min but also open a second tab with our https page. It needed to be in the packaging of it, for the person to install, and open the browser and our page appears in the first time, or then in a second guide along with the tour page. OK, undestanding? Have problem? Thanks!

**Comment by PalmerAL** [comment]
Oh, OK, I didn't realize you were talking about a new installation.

The code that determines what gets opened on startup is here: https://github.com/minbrowser/min/blob/master/js/sessionRestore.js#L20. If you want to always open the same page, you could probably just replace the contents of that function with this:
```
        tasks.setSelected(tasks.add()) // create a new task

        var newTab = tasks.getSelected().tabs.add({
          url: //add URL here
        })
        browserUI.addTab(newTab, {
          enterEditMode: false
        })
```

If you're going to distribute a modified version of Min, please change the name and logo to something else (but do leave a credit in the about dialog/on your website, etc!)

Also, as a user, if I was told to install a specific browser to use your website, I would be kind of annoyed, and I'm not sure I would want to - and Min definitely isn't for everyone, either; I'm guessing some people won't be happy with the design decisions it makes versus more traditional browsers. Depending on what issues you're facing, it might be better to figure out why some browsers are incompatible with your site (or just tell your users to use Firefox or something) rather than forcing everyone to download Min.

**Comment by carlosnizer** [comment]
I will use the min-browser as if it were a desktop app, specific to opening our page that refers to watching the streams. That is, it would be an authentication guide to the server control panel, to access the streams. I think about removing the possibility of opening new guides to other sites. Also remove the imput / change url option from the navigation bar. Understood? Yes, I will certainly keep the credits, also about ... him, the browser, etc. I will make it clear that it was an adaptation to watch our hls streams, showing the benefits, but it will not be mandatory. I will only add our service, for use in the custom min-browser. Although they can also access such service through other browsers. I think I had not made my intention clear ... I know that in this way many functions will no longer be used and will be useless. Who knows in the future I can develop an app for this function.
Thank you very much!

**Comment by carlosnizer** [comment]
I hope you have understood my intention now ...
Maybe, in the future, I will be able to develop a desktop app that has such functions that I need a browser to run streams based on hls. I was quite enthusiastic to understand the electron better.
As per your suggestion, I made the change and everything went well.
I will only include that the app has been changed for our use, keeping the mentions and credits of origin / min-browser.
Finally, as I said, I will remove the options for new tabs and also by clicking on the tab, the function 'imput' / enter new address in the tab. This last function i will  see what I can do. OK?
When everything is ok, I'll send you a link / access to you see and know.
Thank you very much for your attention!

**Comment by PalmerAL** [comment]
So if I understand correctly, you don't actually want to use any of the browser UI, you just want a window that shows your webpage in it?

If that's the case, you probably don't even need anything from Min - you can just use Electron. Take the electron-quick-start app, and replace [these lines](https://github.com/electron/electron-quick-start/blob/master/main.js#L9-L19) with this:

```
  mainWindow = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: {
      nodeIntegration: false
    }
  })

  mainWindow.loadURL(your url here);
```
and everything should work.

**Comment by carlosnizer** [comment]
Very good thanks...
I need to study the electron, it's all new to me.
True, this option you gave me already solves me.
I will see how to edit the apparent menu options. Also, who knows, 
i add option of tabs, for two tabs.
And .... this will be it, that's what I need.
And very good, native, light and functional for what I need.
I got to electron after I got to your browser-min ....
As it is based on html, javascript and css, I believe that I will not have many difficulties. Now i will study the Electron documentation.
Friend, thank you very much for the 'push', for the help.
Thanks.
